### PR TITLE
Feature/form financials

### PR DESCRIPTION
**Pull request by tejue**
Update: I cleaned the code as much as possible. There is still something wrong in calculation when choosing certain products but this is more about calculation than about code I guess.

Because of the code-cleaning, I decided to take the button out for now and will probably reinstall it in a future US.

As a user
I want to see the questions for 'financial issues' as well as answer possibilities
So that I know how much my menstruation costs me

INPUT
- The page shows a form with 6 questions
- The form shows question for 1. which menstruation products are mostly used
- The form accepts choices for: tampons, pad, cup, disc, none
- The form shows question for 2. how many products are in one package (only for tampon and pad)
- The form accepts answer for 1 to 50
- The form shows question for 3. costs of one package
- The form accepts all amounts
- The form shows question for 4. how much taxes are included in one package
- The form accepts amounts from 0 to 30
- The form shows question for 5. possible tax reduction?
- The form accepts "keine Steuer, Teilsteuer, volle Steuer"
- The form shows questions for 6. how often the product is changed per day (only when tampon or pad is selected)- 
The form accepts amounts from 1 to 12

OUTPUT

- The input values are used to calculate the output
- The first output shows 1. the mainly used menstruation product and the amount of money you pay 2. per period, 3. per period-year, 4. in a menstruation-life, 5. until today (2.-5. with the according amount of taxes).
- The page shows an infobox with the content "In Periotopia wären Menstruationsprodukte frei zugänglich. Schottland ist das bisher einzige Land weltweit, in dem Menstruationsartikel in öffentlichen Gebäuden per Gesetz kostenlos bereitgestellt werden müssen. In Deutschland sind es vor allem manche Universitäten, die kostenlose Produkte zur Verfügung stellen. Nur in wenigen Ländern sind Menstruationsprodukte von der Steuer befreit. In Deutschland ist sie seit 2020 zumindest reduziert"
- The page shows a "Periotopia-Index"
- The index is beeing calculated: free = 100%, no tax = 75%, reduced tax = 25%, full tax = 0%
- To get a calculated answer, all questions must be answered
- When a question is not answered an alert appears with the hint, that all fields must be filled out

### CODE REVIEW

File: components/FormFinancials/index.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      >
+        <option>-- Produkt auswählen --</option>
+        <option>Tampon</option>
+        <option>Binde</option>
+        <option>Perioden-Cup</option>
+        <option>Perioden-Disc</option>
+        <option>Perioden-Schlüppi</option>
+      </select>
+      <Question id="packageCosts" question="Wieviel kostet eine Packung?" />
+      <InputNumber
```
Comment (ThysBat) [inline components/FormFinancials/index.js:44]: I saw, that your `InputNumber` Component has the prop `step=1` which means, that you can not enter cents, right? Maybe you want to check on that.

File: components/FormFinancials/index.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          id="taxReduction-none"
+          name="taxReduction"
+          type="radio"
+          value="keine Steuer"
+          checked={taxReduction === "keine Steuer"}
+          onChange={handleFinancials}
+          disabled={packageCosts === "0"}
+        />
+        keine Steuer
+      </label>
```
Comment (ThysBat) [inline components/FormFinancials/index.js:88]: This section was a bit confusing but after a while I guess you wanted to be semantic.
But one label should always be associated with exactly one input. And you actual labels are not connected to the inputs. So what you want here might be a fieldset (Also [here](https://stackoverflow.com/questions/9004307/two-input-fields-inside-one-label) is a link, that clarifies that on stackoverflow).

File: pages/index.js [hunk 3]
```diff
@@ -2,68 +2,147 @@ import { useState } from "react";
...
 
-  function handleMenstruationDaysTillNow(calculatedMenstruationDaysTillNow) {
-    setMenstruationDaysTillNow(calculatedMenstruationDaysTillNow);
+  const {
+    product,
+    packageCosts,
+    taxReduction,
+    taxes,
+    packageContent,
+    changeProdukt,
```
Comment (ThysBat) [inline pages/index.js:52]: A little type, I guess? -> changeProduct

File: components/FormFinancials/index.js (hunk 1 above)
Comment (ThysBat) [reply components/FormFinancials/index.js:44]: Don't mind this, I just found your Vercel Preview and this works just perfect!

**Review by CorneliusMoe — APPROVED** [review]
LGTM :) 
Really amazed by you App so far!

**Pull request merged**